goes.go

```diff
@@ -342,6 +342,27 @@ func (c *Connection) Delete(d Document, extraArgs url.Values) (*Response, error)
 	return r.Run()
 }
 
+// Enable use number behaviour
+func (c *Connection) UseNumberEnable() {
+	c.UseNumber = true
+}
+
+func (req *Request) unmarshalBody(body []byte, esResp *Response) error {
+	if !req.Conn.UseNumber {
+		err := json.Unmarshal(body, esResp)
```

Comment:
For consistency, can you switch this to the same form as below, with the err variable being assigned within the if statement?